EC-84: Restructure eZ Commerce doc, part 2 (#2)
ezsystems/commerce-documentation: docs/guide/product_description/addons_and_plugins.md

The customer center can be connected to the ERP system and synchronizes customer data and contact data when required. In that case the ERP also controls which customer can use the customer center. 

> If a customer has a customer center all employees will be stored beneath the company folder. The shop administrator will have a good overview of the company and its related employees with a shop account. 

Reviewer: ```suggestion
If a customer has a customer center all employees are stored beneath the company folder. The shop administrator has an overview of the company and its related employees with a shop account. 
```

![Rating & Review](img/rating_review_2.png)

### Voucher

> Discount or value vouchers can easily be created in the shop back end. A voucher contains discount conditions like minimum basket value or discountable products. Create individual vouchers that are valid a given number of times for a certain time period or personalized vouchers that are only valid once.

Reviewer: ```suggestion
Discount or value vouchers can easily be created in the shop backend. A voucher contains discount conditions like minimum basket value or discountable products. Create individual vouchers that are valid for a given number of uses, or for a set time period, or personalized vouchers that are only valid for one-time use.
```
